stable/coredns/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: coredns
-version: 0.9.1
-appVersion: 1.0.6
+version: 0.10.0
+appVersion: 1.1.3
```

Comment:
Update to 1.2, now that 1.2 is released. Also request to update the default image tag to 1.2 in Values.yaml